features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelperTest.kt

```diff
@@ -243,4 +276,156 @@ internal class ImageWireframeHelperTest {
         // Then
         assertThat(wireframe).isEqualTo(expectedWireframe)
     }
+
+    @Test
+    fun `M return empty list W createCompoundDrawableWireframes() { no compound drawables }`() {
+        // Given
+        whenever(mockTextView.compoundDrawables)
+            .thenReturn(arrayOf(null, null, null, null))
+
+        // When
+        val wireframes = testedHelper.createCompoundDrawableWireframes(
+            mockTextView,
+            mockMappingContext,
+            0
+        )
+
+        // Then
+        assertThat(wireframes).isEmpty()
+    }
+
+    @Test
+    fun `M return valid start item W createCompoundDrawableWireframes() { with startDrawable }`() {
+        // Given
+        whenever(mockTextView.compoundDrawables)
+            .thenReturn(arrayOf(mockDrawable, null, null, null))
+
+        // When
+        val wireframes = testedHelper.createCompoundDrawableWireframes(
+            mockTextView,
+            mockMappingContext,
+            0
+        )
+        val actualWireframe = wireframes[0] as MobileSegment.Wireframe.ImageWireframe
```

Comment:
minor: I would move `actualWireframe` into the `// Then` section (everywhere in this class)